README.md

```diff
@@ -430,6 +478,19 @@ go run main.go -domain test \
 -descriptors name=foo,age=14 -descriptors name=bar,age=18
 ```
 
+# Global ShadowMode
+
+There is a global shadow-mode which can make it easier to introduce rate limiting into an existing service landscape. It will override whatever result is returned by the regular rate limiting process.
+
+## Configuration
+The global shadow mode is configured with an environment variable
+
+Setting environment variable`SHADOW_MODE` to `true` will enable the feature.
+
+## Statistics
+There is an additional service-level statistics generated that will increment whenever the global shadow mode has overridden a rate limiting result.
```

Comment:
What is the difference between the `over_limit.shadow_mode` and `over_limit.over_limit` metrics? If I'm understanding it correctly, if shadow mode is on for a rate limit, these 2 metrics would always equal. Does it exist so we can observe the shadow testing easier?

Reply:
The reason for having the shadow_mode metrics for global is to be able to distinguish the situation in the metrics series if the configuration is changed over time and that it's not a regular "over_limit". In the global shadow-mode I would like to affect as little as possible "inside" of the service, but want to be able to identify in the metric series that it was not actually ratelimiting due to shadow_mode

Reply:
I can add it to the docker-compose example to make it easy to test

Reply:
Added shadow_mode to the example docker-compose and docs

Reply:
lgtm, thanks!